third_party/forked/golang/expansion/expand_test.go

```diff
@@ -1,8 +1,8 @@
 package expansion
 
 import (
+	api "k8s.io/kubernetes/pkg/apis/core"
```

Comment:
It is recommended that imports are organized in groups

https://github.com/golang/go/wiki/CodeReviewComments#imports

Reply:
@ceshihao fix this by using goimports